core/src/main/java/io/kestra/plugin/core/trigger/Flow.java

```diff
@@ -129,6 +133,25 @@ public Optional<Execution> evaluate(RunContext runContext, io.kestra.core.models
         }
     }
 
+    private List<Label> generateLabels(RunContext runContext, io.kestra.core.models.flows.Flow flow) throws IllegalVariableEvaluationException {
+        final List<Label> labels = new ArrayList<>();
+
+        if (flow.getLabels() != null) {
+            labels.addAll(flow.getLabels()); // no need for rendering
+        }
+
+        if (this.getLabels() != null) {
+            for (Label label : this.getLabels()) {
+                final var value = runContext.render(label.value());
```

Comment:
So best would be try try/catch the exception here, log an error and either exclude the label or add it as is.

Reply:
We discuss it internally, in case of error log it and don't include the label